java/code/src/com/redhat/rhn/domain/action/Action_legacyUser.hbm.xml

```diff
@@ -367,7 +367,7 @@ PUBLIC "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
             <key column="action_id"/>
             <property name="uuid" type="string" column="uuid"/>
             <property name="type" type="string" column="vm_type"/>
-            <property name="name" type="string" column="vm_name"/>
+            <property name="guestName" type="string" column="vm_name"/>
```

Comment:
Don't we need a SQL migration for this one?

Reply:
No, the SQL-side of the mapping hasn't changed: only the Java side